src/header/Heder.stories.tsx

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import {Icon} from "../icon";
+import {Dropdown, DropdownMenu, DropdownItem, MenuItemType} from "../forms/dropdown";
+import {Header, HeaderColor} from "./Header";
+import {Card, CardBlock, CardFooter, CardText, CardTitle} from "../cards";
+
+storiesOf("Header", module)
+    .add("Header Types", () => (
+        <React.Fragment>
+            <div>
+                <h4> Header Types</h4>
+                <Header className={HeaderColor.header6} style={{marginTop: "24px"}}>
+                    <div className="branding">
+                        <a href="#" className="nav-link">
```

Comment:
Try to use NAV and NavLink  components here for all of your stories.

Reply:
Done - used NAV component